src/routes/examples/main/react.mdx

```diff
@@ -1,3 +1,110 @@
 # React
 
-Coming soon!
+Setting up Mirage with Create React App is easy, it only requires a single configuration file.
+
+Let's get started by first installing the `miragejs` package.
+
+```text
+yarn add -D miragejs
+```
+
+or
+
+```text
+npm install --save-dev miragejs
+```
+
+## Defining the server
+
+Create a new file, `src/server.js`, that will hold the code for our mock server. To get started quickly, let's use the following server definition. Paste this code into the new `server.js` file.
+
+```js
+// src/server.js
+
+import { Server, Model, Factory } from "miragejs"
+
+export function makeServer({ environment = "development" } = {}) {
+  let server = new Server({
+    environment,
+
+    routes() {
+      this.namespace = "api"
+
+      this.get("/users", () => {
+        return {
+          users: [
+            {
+              id: 1,
+              name: "Bob",
+            },
+            {
+              id: 2,
+              name: "Alice",
+            },
+          ],
+        }
+      })
+    },
+  })
+
+  return server
+}
+```
+
+This server creates a single endpoint, `/api/users`, that responds with a JSON payload containing two users, Bob and Alice. That's it!
+
+> You might be thinking, why are we creating and exporting a new function `makeServer` instead of just starting the server in this file. There are often times when we want to quickly make new isolated servers, such as when we are writing tests. We recommend every Mirage application use this `makeServer` pattern.
+
+## Starting the server
+
+Now that we have our server defined, let's get it running.
+
+Open your React app's `src/index.js` file, import the `makeServer` function, and invoke it.
```

Comment:
Stylistic nit: I like sticking with either "our" or "your" language (but not both) throughout a tutorial. In the previous sentence we say

> Now that we have our server defined, let's get it running.

which sounds good. Let's preserve the "our" language throughout the document. So this would become something like

> We'll open our React app's `src/index.js` file, import our new `makeServer` function, and invoke it in development: